contributing/code/conventions.rst

```diff
@@ -103,4 +103,10 @@ A PHP ``E_USER_DEPRECATED`` error must also be triggered to help people with
 the migration starting one or two minor versions before the version where the
 feature will be removed (depending on the criticality of the removal)::
 
-    trigger_error('XXX() is deprecated since version 2.X and will be removed in 2.Y. Use XXX instead.', E_USER_DEPRECATED);
+    @trigger_error('XXX() is deprecated since version 2.8 and will be removed in 3.0. Use XXX instead.', E_USER_DEPRECATED);
+
+Without the `@-silencing operator <https://php.net/manual/en/language.operators.errorcontrol.php>`_,
```

Comment:
We don't inline referenced URLs. So in the text you should do something like this:

```rst
Without the `@-silencing operator`_, [...]
```

And then append the reference at the bottom of the file:

```rst
.. _`@-silencing operator`: https://php.net/manual/en/language.operators.errorcontrol.php
```